### PR TITLE
Biometrics settings

### PR DESCRIPTION
**Pull request by ericgibb**

### CODE REVIEW

File: src/screens/Settings/sagas.js [hunk 1]
```diff
@@ -1,30 +1,133 @@
...
+    type: 'Authentication',
+  };
+
+  try {
+    let biometricsResponse = await TouchID.isSupported();
+
+    if (biometricsResponse === 'FaceID') {
+      biometrics.isSupported = true;
+      biometrics.type = 'FaceID';
+    } else {
```
Comment (zachgibb) [inline src/screens/Settings/sagas.js]: does `isSupported` throw an error when support is not available?

File: ios/Hoard/Info.plist [hunk 2]
```diff
@@ -131,5 +131,7 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+  <key>NSFaceIDUsageDescription</key>
+  <string>Enabling Face ID allows you quick and secure access to your account.</string>
```
Comment (zachgibb) [inline ios/Hoard/Info.plist]: was there a touchid one too?

File: src/screens/Settings/sagas.js [hunk 3]
```diff
@@ -1,30 +1,133 @@
...
+  }
+
+  try {
+    let authenticateResponse = await TouchID.authenticate(
+      'To unlock or authenticate Hoard',
+      androidConfigObject
+    );
+
+    // eslint-disable-next-line no-console
+    console.log('authenticateResponse', authenticateResponse);
```
Comment (zachgibb) [inline src/screens/Settings/sagas.js]: if dev

File: src/screens/Settings/sagas.js [hunk 4]
```diff
@@ -1,30 +1,133 @@
...
   while (true) {
-    yield take([
+    const action = yield take([
       // add other settings change events here
       UPDATE_TRADING_PAIR,
+      UPDATE_ENABLE_BIOMETRICS,
       UPDATE_ENABLE_PUSH_NOTIFICATIONS,
     ]);
+
+    if (action.type == 'UPDATE_ENABLE_BIOMETRICS') {
```
Comment (zachgibb) [inline src/screens/Settings/sagas.js]: we could use the constant here

File: ios/Hoard/Info.plist (hunk 2 above)
Comment (ericgibb) [reply ios/Hoard/Info.plist]: nope... Touch ID isn't required in the plist for some reason

File: src/screens/Settings/sagas.js (hunk 4 above)
Comment (ericgibb) [reply src/screens/Settings/sagas.js]: 👍

**Pull request merged**